### ISSUE TITLE
Netlify and Clerk - handshake - too many redirects

### ISSUE THREAD

**Issue by kriss145** [wobsoriano/nuxt-clerk-template#17]
Hi.

I'm using your code from your template in my application to have a Clerk auth.

Unfortunately on Netlify after log-in I have the problem with the message "ERR_TOO_MANY_REDIRECTS" when Clerk redirect me to the application with ?__clerk_handshake param in the query.

During the development all works good, also when I'm running the application locally (in production build).

This problem is on the Netlify. I can't find the problem.

Is any one know how to fix it?

**Comment by wobsoriano** [comment]
hey! thank you for reporting. I myself haven't tried deploying a Nuxt app in Netlify.

I will deploy a clone of this repo to Netlify and let you know what I'll find.

**Comment by wobsoriano** [comment]
So I deployed a copy of this to netlify and was able to replicate the infinite redirects - https://nuxt-clerk-template.netlify.app/

- first visit, infinite redirect
- close it, and open it again, it works

**Comment by kriss145** [comment]
Yes, so it is Netlify problem? I think that somewhere there is the problem. Do u know how to fix that?

**Comment by qwerzl** [comment]
Same here. Any insights on this?

**Comment by wobsoriano** [comment]
Still trying to figure how we can make netlify not cache a certain route with query param

**Comment by johnnydecimal** [comment]
Just checking in, I have this problem. Does Netlify not really work at the moment? No problem if not, I’ll use Vercel.

**Comment by wobsoriano** [comment]
> Just checking in, I have this problem. Does Netlify not really work at the moment? No problem if not, I’ll use Vercel.

This bug appears only in Clerk dev instance in Netlify. I'm still looking for a way to fix it 😕

**Comment by johnnydecimal** [comment]
Oh interesting. So if I’m on a prod instance I should be okay? All of my other stuff is on Netlify so I’d prefer to keep everything there.

Good luck! I appreciate your work. This is way above my skill level and I am in awe. :)

**Comment by wobsoriano** [comment]
> Oh interesting. So if I’m on a prod instance I should be okay? All of my other stuff is on Netlify so I’d prefer to keep everything there.
> 
> 
> 
> Good luck! I appreciate your work. This is way above my skill level and I am in awe. :)

Thanks and yes, works 💯 in prod instance of Clerk.
